go-controller/pkg/cni/helper_linux.go

```diff
@@ -259,7 +259,8 @@ func (pr *PodRequest) ConfigureInterface(namespace string, podName string, ifInf
 		"interface", hostIface.Name,
 		fmt.Sprintf("external_ids:attached_mac=%s", ifInfo.MAC),
 		fmt.Sprintf("external_ids:iface-id=%s", ifaceID),
-		fmt.Sprintf("external_ids:ip_address=%s", ifInfo.IP),
+		// DUAL-STACK FIXME: handle multiple IPs (or remove external_ids:ip_address?)
```

Comment:
@danwinship yeah we don't actually use it, but it might be of debugging use. I think we can kill it though.

Reply:
Well, if it's for debugging we could just change it to be `ip_addresses` and have a comma-separated value. Or we could declare that it only shows the first IP, if that's sufficient for debugging...

Reply:
updated to do `ip_addresses`